lib/thesis/stores/store.ex

```diff
@@ -25,4 +25,11 @@ defmodule Thesis.Store do
       update(%{"slug" => "/"}, %{"Heading" => "My Heading Content"})
   """
   @callback update(%{String.t => String.t}, map) :: atom
+
+  @doc """
+  Deletes the given page (identified by its slug). Returns `:ok`.
+
+      delete(%{"slug" => "/asdf"})
+  """
+  @callback delete(%{String.t => String.t}) :: atom
```

Comment:
Added one more required function for the `Thesis.Store` behaviour (like an interface).